zgw/catalogi-api/src/main/kotlin/com/ritense/catalogiapi/CatalogiApiPlugin.kt

```diff
@@ -77,15 +79,24 @@ class CatalogiApiPlugin(
         @PluginActionProperty statustype: String,
         @PluginActionProperty processVariable: String,
     ) {
-        val statustypeUrl = if (statustype.matches(HTTPS_REGEX)) {
-            statustype
-        } else {
-            val document = AuthorizationContext.runWithoutAuthorization { documentService.get(execution.businessKey) }
-            val zaaktypeUrl = zaaktypeUrlProvider.getZaaktypeUrl(document.definitionId().name())
-            getStatustypeByOmschrijving(zaaktypeUrl, statustype).url!!.toASCIIString()
-        }
+        withLoggingContext(
+            CATALOGI_API.STATUSTYPE to statustype
+        ) {
+            logger.debug { "Retrieving statustype: $statustype and storing it in process variable: $processVariable" }
+
+            val statustypeUrl = if (statustype.matches(HTTPS_REGEX)) {
+                statustype
+            } else {
+                val document =
+                    AuthorizationContext.runWithoutAuthorization { documentService.get(execution.businessKey) }
+                val zaaktypeUrl = zaaktypeUrlProvider.getZaaktypeUrl(document.definitionId().name())
+                getStatustypeByOmschrijving(zaaktypeUrl, statustype).url!!.toASCIIString()
+            }
+
+            logger.info { "Setting process variable $processVariable with statustype URL: $statustypeUrl" }
```

Comment:
Same issue as above i think. This says that some variable has been set, but not really why. I think the user wants to know that this is based on the retrieval.